core/src/main/resources/defaults/blueprints/hdp26-edw-analytics-shared.bp

```diff
@@ -104,10 +104,6 @@
           "hive.merge.mapfiles": "true",
           "hive.server2.tez.initialize.default.sessions": "true",
           "hive.server2.transport.mode": "http",
-          "hive.server2.authentication.ldap.url":"{{ LDAP_URL }}",
-          "hive.server2.authentication.ldap.Domain":"{{ LDAP_DOMAIN }}",
-          "hive.server2.authentication.ldap.baseDN": "{{ LDAP_SYNC_SEARCH_BASE }}",
-          "hive.server2.authentication":"LDAP",
```

Comment:
Need to remove the corresponding configs from hive-interactive-site as well:
https://github.com/sequenceiq/cloudbreak/blob/master/core/src/main/resources/defaults/blueprints/hdp26-edw-analytics-shared.bp#L72